app/controllers/admin/questions_controller.rb

```diff
@@ -1,17 +1,16 @@
 module Admin
   class QuestionsController < Admin::BaseController
     load_and_authorize_resource :conference, find_by: :short_title
-    load_and_authorize_resource through: :conference, except: [:new, :create]
+    load_and_authorize_resource except: [:create]
 
     def index
       authorize! :index, Question.new(conference_id: @conference.id)
-      @questions = Question.where(global: true).all | Question.where(conference_id: @conference.id)
-      @questions_conference = @conference.questions
-      @new_question = @conference.questions.new
+      @questions = Question.where(global: true).all | Question.where(conference_id: @conference.id) | @conference.questions
```

Comment:
What's the difference between 
``Question.where(conference_id: @conference.id) ``
and 
``@conference.questions``